client/starwhale/consts/__init__.py

```diff
@@ -1,5 +1,7 @@
 import pathlib
+import threading
 
+context_holder = threading.local()
```

Comment:
Do we move `context_holder` to `starwhale.api._impl.job`?  `starwhale.consts` module stores the unchanged const.

Reply:
done